README.md

```diff
@@ -1,31 +1,403 @@
+10. Remove the `moduleFederationPlugin` and replace with the `DynamicRemotePlugin`: 
+```jsx
+// declare shared dependencies
+
+const { DynamicRemotePlugin } = require('@openshift/dynamic-plugin-sdk-webpack');
+
+const sharedModules = {
+    '@openshift/dynamic-plugin-sdk':( singleton: true},
+    '@scalprum/react-core':( singleton: true},
+    react: { singleton: true),
+    'react-dom': ( singleton: true },
+
+const dynamicPlugin = new DynamicRemotePlugin({
+    extensions: [],
+    sharedModules,
+    entryScriptfilename:'remoteModule.(fullhash].js'
+});
+
+```
+11. In the `webpack.config.js` file, also replace the `moduleFederationPlugin` with the `dynamicPlugin` plugins: 
+
+```jsx
+   plugins: [
+
+    new MiniCssExtractPlugin(),
+    // scalprum required plugin
+    dynamicPlugin
+    ,
+  ],
+```
+12.  In the `webpack.config.js` file, also add a `publicPath`: 
+```jsx
+const config = {
+    entry: "./src/index.tsx",
+    output: {
+    path: path.resolve(_dirname, "dist"),
+    publicPath:'http://localhost:8003'
+    },
+devServer: {
+    open: true,
+    host: "localhost",
+},
+
+```
+
+13. In your **scalprum-demo/remoteModule** directory, create a **src** directory. 
+14. In your **scalprum-demo/remoteModule/src** directory, create an `index.tsx` file. 
+15. In your **scalprum-demo/remoteModule/src** directory, create a `RemoteModuleComponent.tsx` file and add the following: 
+```jsx
+import React from 'react';
+
+const RemoteModuleComponent = () => {
+  return (
+    <div>
+    I am a remote module component;
+    </div>
+  )
+}
+
+export default RemoteModuleComponent; 
+
+```
+16. In your **scalprum-demo/** root directory, create a `plugin-metadata.json` file, and add this example text: 
+```jsx
+{
+  "name": "remoteModule",
+  "version": "1.0.0",
+  "exposedModules": {
+    "RemoteModuleComponent": "./remoteModule/src/RemoteModuleComponent.tsx"
+  },
+  "extensions": []
+}
+```
+Here you define what components you are exposing.
+The value `RemoteModuleComponent` is defined in the `ScalprumRoot.tsx`. The `"./remoteModule/src/RemoteModuleComponent.tsx"` points to the file's location. 
+
+17. In the `scalprum-demo/package.json` file, add the following entry and point to `remoteModule/webpack.config.js` as your configuration file: 
+
+```json 
+
+"build:plugin":"webpack --mode=production --node-env=production -c remoteModule/webpack.config.js"
+
+```
+
+
+To test if everything is set up correctly, enter the following command: 
+
+```shell 
+$ npm run build:plugin
+```
+
+Normally, your modules would be served via your own content delivery network, but for demo purposes, you can serve it locally from the **dist** directory that is created when you run `$ npm run build:plugin`.
+
+```shell
+$ scalprum-demo/remoteModule/dist/ && npx http serve . -p 8003
+```
+
+If this action completes without error, you can view your plugin manifest at `http://localhost:8003/`. 
+
+
```

Comment:
We should probably also say that they should run the host application via `npm run serve` and check if the dynamic plugin was loaded correctly in the UI.